dradis/dradis-ce
# Port Boards: Summary widget

### Spec

Port methodologies summary widget in the project summary page to use the new methodologies format (boards)

### How to Test
Add a board with lists and tasks.
Check the project summary page.
Assert we have a nice donut chart.
Also assert that before the chart loads, we see a loading spinner in its place.

### Copyright assignment

> I assign all rights, including copyright, to any future Dradis
> work by myself to Security Roots.

### Check List

~~- [ ] Added a CHANGELOG entry~~
